tasks/package.js

```diff
@@ -17,8 +17,14 @@ module.exports = function(grunt) {
       srcFiles = ['**', '!**/.gitkeep'].concat((config && config.srcFiles && config.srcFiles.length) ? config.srcFiles : '**'),
       projFiles = (config && config.projFiles && config.projFiles.length) ? config.projFiles : [];
 
-    var destPath = grunt.config.get('config.buildPaths.package') + '/package';
-    var tasks = []
+    // Look for a package target spec, build destination path.
+    var packageTarget = grunt.option('target') || process.env.GRUNT_PACKAGE_TARGET;
```

Comment:
Revisiting this now, as I think the package stack will get some attention soon. The environment variable isn't a bad fallback, but each project is likely to have a single target needed ubiquitously. I think Gruntconfig.json overridden by a command-line flag will have much more use.

```
var packageTarget = grunt.option('target') || grunt.config.get('config.package.target');
```

In looking at the UI, I also wonder if package --name or --path might be a more intuitive naming convention.